src/resource_server/utils/commands.json

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
this is deployment config and should probably not be within package source (can be within this repo, but not as python pkg resource)